ios/Podfile

```diff
@@ -56,24 +58,28 @@ target 'JitsiMeetSDKLite' do
 
     # React Native and its dependencies
     #
-  
-    config = use_native_modules!
+
+    $config = use_native_modules!
     use_react_native!(
-      :path => config["reactNativePath"],
```

Comment:
Why is the path missing on the Lite target but not on the full one?

Reply:
Oops. Mistake here. Will re-add it.